src/core/agent/services/signifyNotificationService.ts

```diff
@@ -557,9 +565,46 @@ class SignifyNotificationService extends AgentService {
                   .get(admitExchange.exn.p);
                 const credentialId = grantExchange.exn.e.acdc.d;
                 if (credentialId) {
-                  await Agent.agent.ipexCommunications.markAcdcComplete(
-                    credentialId
+                  await Agent.agent.ipexCommunications.markAcdc(
+                    credentialId,
+                    CredentialStatus.CONFIRMED
+                  );
+                }
+              }
+              break;
+            }
+            case OperationPendingRecordType.ExchangeRevokeCredential: {
+              const admitExchange = await this.props.signifyClient
+                .exchanges()
+                .get(operation.metadata?.said);
+              if (admitExchange.exn.r === ExchangeRoute.IpexAdmit) {
+                const grantExchange = await this.props.signifyClient
+                  .exchanges()
+                  .get(admitExchange.exn.p);
+                const credentialId = grantExchange?.exn?.e?.acdc?.d;
+                const credential = await this.props.signifyClient
+                  .credentials()
+                  .get(credentialId);
+                if (credential && credential.status.s === "1") {
+                  await Agent.agent.ipexCommunications.markAcdc(
+                    credentialId,
+                    CredentialStatus.REVOKED
                   );
+                  const metadata: any = {
+                    id: uuidv4(),
+                    a: {
+                      r: grantExchange.exn.r,
```

Comment:
OK, I think the best thing to do here is we do something like this:

```
enum NotificationRoute {
  // "Real" notifications from KERIA
  ExnIpexGrant = "/exn/ipex/grant",
  MultiSigExn = "/multisig/exn",
  MultiSigIcp = "/multisig/icp",
  MultiSigRot = "/multisig/rot",
  MultiSigRpy = "/multisig/rpy",
  ExnIpexApply = "/exn/ipex/apply",
  ExnIpexAgree = "/exn/ipex/agree",

  // Notifications from our wallet to give further feedback to the user
  LocalAcdcRevoked = "/local/acdc/revoked",
}
```

The function `markNotification` should be updated to ignore routes that start with `/local`

I think this would work.

Reply:
`private async markNotification(notiSaid: string) {
    return this.props.signifyClient.notifications().mark(notiSaid);
  }`
  
  This is the inside of `markNotification`, I think we can't ignore the route here, right?

Reply:
ah yeah, lets do it in `deleteNotificationRecordById` and update `deleteNotificationRecordById` to accept the route too